docker/bpe/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ghcr.io/num-codex/highmed-dsf/bpe:25adc6485d2b09c35b5d2b85f4fd0ece13b214d5
```

Comment:
Can we use the M1 image here?

Reply:
M1-Image has been superficially checked in combination with the develop project. No issues detected so far and the process is also running.
Thus, switched to M1.